.drone.yml

```diff
@@ -64,3 +64,24 @@ pipeline:
     when:
       branch: master
       event: push
+
+  deploy_production_cronjobs:
+    group: deploy
+    image: quay.io/mongodb/drone-helm:v2.14.1-0.1.0
+    # The release name should be unique across the namespace, the app or repo name is recommended
+    release: selected-tests
```

Comment:
I think this is supposed to be different from the web app above and unique in the namespace.